packages/util/src/types.ts

```diff
@@ -118,3 +118,20 @@ export function toType<T extends TypeOutput>(
       throw new Error('unknown outputType')
   }
 }
+
+type Falsy = false | '' | 0 | null | undefined
+
+export function isFalsy(value: unknown): value is Falsy {
+  return !!(
+    value === false ||
+    value === '' ||
+    value === 0 ||
+    value === null ||
+    typeof value === 'undefined' ||
+    value === BigInt(0)
+  )
+}
+
+export function isTruthy<T>(value: T | Falsy): value is T {
+  return !isFalsy(value)
+}
```

Comment:
Also, on another note: are you confident that this code is correct, since we would be using this all over the place now in our libraries (for now)? So does this match 1:1 the previous behavior?

I guess we very much would want to have some tests (or rather: somewhat complete, type-wise) on this, to be really sure this behaves correctly in all cases.

Reply:
Yeah I am confident, but can definitely add tests since as you say this is used throughout

Reply:
Thanks, yeah, guess this should be some really compact LoCs to cover the various cases.

Reply:
Added some tests